### PR TITLE
Seo keywords metadata

### PR DESCRIPTION
**Pull request by query-string**
Related to issue #12

### CODE REVIEW

**Review by FanaHOVA — APPROVED** [review]

File: source/layouts/layout.slim [hunk 1]
```diff
@@ -6,6 +6,9 @@ html
     meta charset="utf-8" /
     meta content="IE=edge;chrome=1" http-equiv="X-UA-Compatible" /
     meta content="width=device-width, initial-scale=1" name="viewport" /
+    meta content="SABEResPODER te ayuda prosperar en los EEUU, con recursos y contestas personalizadas a tus preguntas sobre inmigración, educación, finanzas, salud y otros asuntos importantes. Tu causa es nuestra causa!" name="description" /
+    meta content="Inmigracion, ciudania, preguntas, articulos, derechos, finanzas, salud, educacion, comunidad, consulado, visa, pasaporte, estados unidos, informacion, ofertas, viviendo en los estados unidos, trabajar, trabajo, impuestos, taxes, embajada, el salvador, mexico, guatemala, informacion gratuita, medicamentos, farmacias, farmasep, descuentos, encuestas" name="keywords" /
```
Comment (dvdbng) [inline source/layouts/layout.slim]: Several typos in this line: ciuda**da**nia, art**í**culos, educaci**ó**n, informaci**ó**n, also I would add EEUU in addition to united states

Comment (query-string) [reply source/layouts/layout.slim]: Thanks David. It is interesting because in #12 @stan2020 listed keywords exactly in this way...

Comment (dvdbng) [reply source/layouts/layout.slim]: @query-string well, anyone can make typo :) and I guess typing those accents in an American keyboard is a PITA

**Comment by stan2020** [comment]
Thanks for the fixes, guys.  Also, Raul has pointed out to me that "respuestas" is better than "contestas".  When you get a chance, please change the meta content from "... con recursos y contestas personalizadas..." to "...con recursos y respuestas personalizadas...".

**Pull request merged**